src/components/Search/SearchAutocomplete/SearchAutocomplete.style.tsx

```diff
@@ -55,12 +55,14 @@ const MobileWrapperClosed = css`
   height: unset;
   width: ${props => props.theme.searchbar.mobileClosedWidth};
   padding: ${props => props.theme.searchbar.mobileClosedPadding};
-  margin-left: ${props => props.theme.searchbar.mobileClosedMarginLeft};
   max-width: ${props => props.theme.searchbar.mobileClosedMaxWidth};
+  margin: auto;
 `;
 
 export const Wrapper = styled.div<{ $isOpen: boolean }>`
   ${({ $isOpen }) => ($isOpen ? MobileWrapperOpened : MobileWrapperClosed)};
+  width: 100%;
+  max-width: ${mobileWidth}px;
```

Comment:
this^ results in the menu not being full bleed on mobile:

<img width="377" alt="Screen Shot 2021-05-17 at 6 39 36 PM" src="https://user-images.githubusercontent.com/44076375/118564948-4eee6100-b73f-11eb-8890-f5a1d715f47d.png">

Reply:
nice catch, thanks.  removing both is fine, I didn't need them after all, so long as I change the 'closed' one in the theme from '350px' to '100%';